### PR TITLE
Restore ACT branding & navbar

### PR DESCRIPTION
**Pull request by WilcoFiers**
This pull request makes a few changes:
1. Update the title from "WCAG 2 Test Rules" to "Accessibility Conformance Testing (ACT)" and makes it into a link
2. Add a navbar with All Rules, About, and Test Tools & Methodologies
3. Remove the "About WCAG 2 Test Rules" link from the head (it is in the navbar now)
4. Updates the tagline from "For developers of evaluation tools and test methodologies" to "Harmonised rules for WCAG 2 & ARIA"
5. Renames "Test Rules" to "ACT Rules", clarifying that ACT also covers non-WCAG things in further texts.
6. Renames the Test Rules list to "Rules for WCAG 2"
7. Split the Proposed rules list up into "Proposed Rules for WCAG 2" and "Other Proposed ACT Rules"
8. Various editorial changes in All ACT Rules, and About ACT Rules
9. Do more cross-linking to various places
10. Flag Rule list & About pages as draft, since this hasn't been OK'ed by AG / W3C before

There is no "Approved non-WCAG rules" list, because there are none. In case you're wondering. We're going to have ARIA rules at some point, but those aren't ready yet.

### CODE REVIEW

**Comment by SteveALee** [comment]
@wilco could you please add a brief description identifying the requirement and outlining the approach? Thanks

**Comment by shawna-slh** [comment]
@WilcoFiers , A succinct list of proposed changes in the PR would be helpful -- some of us can process that much easier than the files changed diffs, thus we could reply faster.

<hr>

There is support for some of this. I am working on consistency across related designs/resources. I hope to get back to this specific proposal next week. Thanks for your patience in the meantime.

**Review by colabottles — APPROVED** [review]

**Comment by SteveALee** [comment]
@WilcoFiers thanks for adding the details of this PR, that's really helpful.

Is it now ready for review after the flurry of commits?

**Review by SteveALee — APPROVED** [review]
Looks good from the template perspective. Over to @shawna-slh for the rest

**Comment by wcreedle** [comment]
I'm not a reviewer but it looked good to me

**Comment by tombrunet** [comment]
At a high level, this looks good. Some minor things:
* I did notice some spelling issues, but didn't exhaustively note them. One example is "therefor". 
* Also, does it matter British vs US spelling (e.g., harmonisation vs harmonization)?
* The `#understandingactconsistency` link doesn't seem to work

**Review by daniel-montalvo — COMMENTED** [review]
This is great work! Just some comments and suggestions for consideration.

File: _config.yml [hunk 1]
```diff
@@ -54,19 +54,21 @@ defaults:
...
+        rule_type: "Atomic rule"
       sidebar: true
       standalone_resource_header:
-        title: WCAG 2 Test Rules
-        subtitle: For developers of evaluation tools and test methodologies
-        link:
-          title: About WCAG 2 Test Rules
+        title: Accessibility Conformance Testing (ACT)
+        title-ref: /standards-guidelines/act/rules/
+        subtitle: Harmonised rules for WCAG 2 & ARIA
```
[the remark this answers is not in the document]
Comment (SteveALee) [reply _config.yml:64]: Not W3C and/or EO or WAI have prefered. I think US style English. But I also think the Oxford English Dictionary decided Z is ok too some while ago.

@shawn might know

File: _config.yml [hunk 2]
```diff
@@ -54,19 +54,21 @@ defaults:
...
+      standalone_resource_nav_links:
+        - name: All Rules
+          ref: /standards-guidelines/act/rules/
+        - name: About
           ref: /standards-guidelines/act/rules/about/
-      standalone_resource_pager:
-        name: All Test Rules
-        ref: /standards-guidelines/act/rules/
+        - name: Test Tools & Methodologies
+          ref: /standards-guidelines/act/implementations/
```
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply _config.yml:71]: Thanks, and you are right. You suggested "matrix", but since Shawn suggested she wanted to see something else I'm waiting for that "something else" before making a change we may change again next week.

**Comment by shawna-slh** [comment]
I finally cleared some time and headspace to put on a few user hats, and WAI outreach and website coordinator hats. I don’t know enough about frequent ACT Rules users to be able to wear that hat. A couple hats I could wear: 1. someone who lands on these pages without any context. 2. someone wanting to learn more about specific tools that they might want to use.

Notes:
* apologies that my comments aren’t more polished and succinct; I ran way over time on this
* I didn’t read all of the text, and I didn’t read all of the information in this pull request
* Wilco asked for me to put all comments in this pull request, and not create separate issues for different points

So here goes:

**Comment by shawna-slh** [comment]
[rationale] Primary user needs that we addressed with this redesign include:
* Make it clear what the page is, e.g., for users who land on a page from a search engine
* Make it clear that it’s not required to meet WCAG (e.g., https://github.com/w3c/wai-minimal-header-design/issues/4)

[suggestion] Based on that, I think it’s very important that the banner include ‘For developers of test tools and methodologies’. **Other users need to know that this detailed, technical mumbo jumbo is not something they need to try to wade through and understand.** Also, I think if that’s in the banner, then we don’t have to have ‘not required to meet WCAG’ in the banner like we do for Understanding, Techniques, and Supplemental Guidance (provided it’s elsewhere on the pages, per below).

[rationale] Probably every page, including every test rule, needs to say that it is not required for conformance to WCAG. That is the case for Techniques e.g., https://github.com/w3c/wai-minimal-header-design/issues/4 For the Rules, perhaps the following is sufficient:

[suggestion] for clearly communicating not required wherever people land:
* In About page Summary box, leave ‘ACT Rules are informative — that means they are not required for determining conformance to WCAG.’
* In All ACT Rules Summary box, add: ‘ACT Rules are informative — that means they are not required for determining conformance to WCAG.’
* To all rules pages, add: ‘ACT Rules are informative — that means they are not required for determining conformance. The basis for determining conformance to WCAG is the success criteria from the WCAG standard — not the ACT Rules.’ Probably this is best added with heading of ‘ACT Rules are Informative’ at the end of the main conten, right before the ‘Help improve this page’ box. Possibly it would be sufficient in the page footer, before the Rule Identifier.

[rationale] Put on the hat of a designer or developer looking for guidance on XYZ... From a search engine you land on an ACT Rule. Most will probably be overwhelmed and just leave the page. Some will click on About. >switch hats< Won’t you please help them find their way to more relevant resources? Since these Rules use the ‘minimal header’ without all the WAI site navigation, a hand out would be appreciated.

[suggestion] In the About Summary box, leave this existing content from  https://www.w3.org/WAI/standards-guidelines/act/rules/about/
> ACT Rules are primarily for developers of evaluation tools and test methodologies.
> Guidance for others is in <cite>Understanding WCAG</cite> and <cite>WCAG Techniques</cite>. **Learn about the different WCAG support material from [The WCAG 2 Documents](https://www.w3.org/WAI/standards-guidelines/wcag/docs/)**.

[background] One reason for changing the banner (essentially rebranding from ‘WCAG 2 Test Rules’) was because some people are familiar with ‘ACT Rules’. However, I don’t see ‘ACT Rules’ in the first two lines at all right now.

[suggestion] Here’s an approach to having a succinct catchy banner yet still writing out ACT soon:
* change banner text from ‘Accessibility Conformance Testing (ACT)’ to ‘ACT Rules’
* change navigation text from ‘All Rules’ to ‘All ACT Rules’
* probably change navigation text from ‘About’ to ‘About Accessibility Conformance Testing (ACT) Rules’ and make it the last item in the nav
* spell out ‘Accessibility Conformance Testing (ACT) Rules’ as soon as feasible in the summary and/or main content

**Comment by shawna-slh** [comment]
**[!important] WCAG and ARIA and not required for conformance**

[rationale] Many people do not understand the relationship between WCAG and ARIA. I’ve heard people think ARIA is a replacement for WCAG. I’ve heard people – who have spent a lot of time and effort trying to understand what they are required to do legally -- think that they have to implement all of ARIA in order to conform to WCAG.

It’s important that this Rules resource helps avoid misunderstandings, and does not further contribute to confusion.

[suggestion] Therefore, I suggest editing: “This page contains list of ACT Rules to test conformance Web Content Accessibility Guidelines (WCAG), WAI-ARIA and other accessibility practices.” and “…for determining conformance to WCAG or ARIA.”

---

More broadly, I wonder about calling out ARIA at all? We talked about having “Rules for WCAG 2” and “Rules for ARIA”; however, I’m not sure if that’s needed or useful or do-able or the right way to present it? I note that ‘ARIA required context role’ and ‘ARIA required owned elements’ are to test WCAG ‘1.3.1: Info and Relationships’. If there is a list of WCAG 2 and a list of ARIA, it’s not clear where and why they go in which list.

Oh, now I notice a new section has been added “Other Proposed ACT Rules” that says: These ACT Rules are not required for conformance to WCAG. They are part of various other accessibility standards and best practices, such as WAI-ARIA and Techniques for WCAG 2 .

-- I scroll up and see that new text has been added to the previous sections: 
> These ACT Rules are used to test conformance issues of WCAG 2.0 or 2.1.

This seems to conflict with “ACT Rules are informative — that means they are not required for determining conformance.” stated elsewhere (which is correct).

Here is some wording that we’re used elsewhere to differentiate such information:
* [The WCAG Documents](https://www.w3.org/WAI/standards-guidelines/wcag/docs/#supporting-documents):  … The supporting documents directly relate to WCAG guidelines and success criteria. They are not required to meet WCAG; they are “informative” or “non-normative”. … Supplemental guidance goes beyond the requirements of WCAG. It is not required to meet WCAG. Much of the guidance is essential…
* [media resource](https://www.w3.org/WAI/media/av/av-content/#introduction) Some of the guidance below is related to requirements in Web Content Accessibility Guidelines (WCAG)... Other guidance is good practice.

[suggestion] Perhaps this grouping and phrasing would work:
* **Rules for WCAG 2**<br>The ACT Rules in this section directly relate to WCAG success criteria.
* **Rules beyond WCAG**<br>The Act Rules below do _not_ directly relate to WCAG success criteria. They relate to other accessibility guidance such as WAI-ARIA or WCAG Techniques.

**Comment by shawna-slh** [comment]
Ref: Test Tool & Methodology Matrix https://deploy-preview-103--wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/

What is the purpose of the page? Who is the audience? What are users trying to accomplish that this page helps them with? How does this relate to the long-standing Eval Tools List ([(existing](https://www.w3.org/WAI/ER/tools/), [redesign draft](https://master--wai-evaluation-tools-list.netlify.app/list-of-evaluation-tools/))?

I would need at least “quick and dirty” info above to be able to provide good input. Here are a few thoughts regardless of above.

---

While I appreciate that “matrix” was the term used in the project proposal, I don’t think this is a matrix, I don’t think matrix communicates what it is, and I don’t think we are bound to using that term.

I think the most accurate title is something like “ACT Rules Implementations”. I seem to recall someone thought that might not be well understood? What if that is part of a longer title? Could have title and subtitle, or long title with short version in the navigation. Also note that we have the Web Accessibility Evaluation Tools List and soon a Digital Accessibility Course List and Accessible Authoring Tools List – so possibly “List” would be good to include? Some brainstorms:
* title: ACT Rules Implementation in Test Tools and Methodologies<br>nav: Implementations in Tools and Methodologies
* title: ACT Rules Implementation Reports<br>subtitle: List of accessibility test tools and methodologies that use ACT Rules<br>nav: ACT Rules in Test Tools and Methodologies

---
[minor] For title, I prefer writing out “and” – because it’s easier to understand and more formal than “&”, and just looks better to me :-). For navigation, OK with “&”, though I think it’s a tad jarring. I don’t feel strongly at all.

---
[medium] Report column cells missing affordance that the image/icon is clickable. Maybe make it look like a button?

---
[medium-minor] Why the order:
* Test Methodologies
* Semi-automated Test Tools
* Automated Test Tools

?
I’m guessing Test Methodologies will always have the least number of listings with the least number of consistent rules. And Automated Test Tools will have the most of both. Also, the wording is usually “test tools and methodologies”. Therefore, maybe better to put Automated Test Tools first and Test Methodologies last?

---
[minor] “WCAG 2.1 all levels” -> “WCAG 2.1 Level A, AA, AAA”
and to match, “WCAG 2.0 Level A & AA” -> “WCAG 2.0 Level A, AA”

---
[minor]  copyedit
> The tables on these pages show how many ACT Rules different accessibility test tools and methodologies have consistently implemented. Each implementation links to a report with more details on how rules are implemented. See understanding ACT consistency for details.

… I was going to suggest copy edit – but I ran out of steam, sorry. For now, I’ll just say: consider quotes around ‘consistently implemented’ in the first sentence since that is a term specific to this.

---
[medium] Add a Tool or Methodology button
It is important that it be clear that we *actively invite* people to submit implementation reports. I understand that many not will. Yet the *perception* of the invitation is important. For that reason, I think it would be good to make the invitation strong and easier. On the other Lists, we have a button to submit.

---

[medium] email contact
> …  If there are any questions or concerns about the information on these pages, open an issue on GitHub or e-mail wai@w3.org.

Probably better to set up a specific e-mail list that goes to multiple people who are in a position to address it (rather than relying on me to forward it :-).

**Comment by SteveALee** [comment]
Agree matrix is abstract and better to call out the exact purpose.

File: content/about.md [hunk 3]
```diff
@@ -11,68 +12,77 @@ github:
...
+{::nomarkdown}
+{% include box.html type="start" title="Summary" class="" %}
+{:/}
 
-Guidance for others is in <cite>Understanding WCAG</cite> and <cite>WCAG Techniques</cite>. **Learn about the different WCAG support material from [The WCAG 2 Documents {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/standards-guidelines/wcag/docs/).**
+ACT Rules describe how to test conformance of accessibility standards such as [Web Content Accessibility Guidelines (WCAG) {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/standards-guidelines/wcag/) and [WAI-ARIA {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/ARIA/). ACT Rules inform accessibility testers on how to evaluate edge cases in way that is consistent with other accessibility testers, test tools, and methodologies. ACT Rules are informative — that means they are not required for determining conformance to WCAG or ARIA.
```
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply content/about.md:23]: I'm not sure about this one... consistency isn't a meaningful term unless you say what it is consistent with. The goal of ACT is to harmonise WCAG testing; to get greater consistency between testers; to improve inter-rater reliability if you will. By not saying "other tools and methodologies" I think it makes it sound like testers aren't consistent with themselves. That's inner-rater reliability, not inter-rater reliability. That's not what we're doing.

The goal is improved consistency between different testers. I think we have to say that. People won't know that's what we're doing if we don't say so.

File: content/about.md [hunk 4]
```diff
@@ -11,68 +12,77 @@ github:
...
+ACT Rules describe how to test conformance of accessibility standards such as [Web Content Accessibility Guidelines (WCAG) {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/standards-guidelines/wcag/) and [WAI-ARIA {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/ARIA/). ACT Rules inform accessibility testers on how to evaluate edge cases in way that is consistent with other accessibility testers, test tools, and methodologies. ACT Rules are informative — that means they are not required for determining conformance to WCAG or ARIA. 
 
 {::nomarkdown}
 {% include box.html type="end" %}
 {:/}
 
-The [List of Test Rules for WCAG 2](https://www.w3.org/WAI/standards-guidelines/act/rules/) is updated periodically. The rules are developed according to the [Accessibility Conformance Testing (ACT) Rules Format 1.0 {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/standards-guidelines/act/) standard.
+## What are ACT Rules
```
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply content/about.md:29]: I didn't think so. Looking on other WAI pages, even when headings are questions they don't include question marks. Here's an example: [What is accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/#what)

**Comment by WilcoFiers** [comment]
@shawn, thank you for the review. I believe I've addressed everything that should be addressed. For the future, would you mind using Github's inline comment feature when it comes to comments on specific parts. That way I can address / comment on them one at a time.

There are a couple of comments that I didn't address. Motivation for doing so below:

> - In About page Summary box, leave ‘ACT Rules are informative — that means they are not required for determining conformance to WCAG.’

I don't think that we changed. Am I misunderstanding your comment, or did you miss this?

> - To all rules pages, add: ‘ACT Rules are informative — that means they are not required for determining conformance. The basis for determining conformance to WCAG is the success criteria from the WCAG standard — not the ACT Rules.’ Probably this is best added with heading of ‘ACT Rules are Informative’ at the end of the main conten, right before the ‘Help improve this page’ box. Possibly it would be sufficient in the page footer, before the Rule Identifier.

This used to be in the sidebar. Steve removed this when he updated how the ToC works. I don't know why that decision was made, but I would like to see it reversed. I'll open an issue for this. This is not related to this PR.

> - In the About Summary box, leave this existing content from https://www.w3.org/WAI/standards-guidelines/act/rules/about/

The summary was just reworded. The only thing that was removed was the link to different WCAG support materials. I do not think something like that belongs in a summary. It's a summary of the current page after all. There is now a section with related resources which is more extensive than we can reasonably put in a summary.

> - I suggest editing: “This page contains list of ACT Rules to test conformance Web Content Accessibility Guidelines (WCAG), WAI-ARIA and other accessibility practices.” and “…for determining conformance to WCAG or ARIA.”

ACT Rules can not be used to determine conformance. They can be used as part of testing conformance, but they aren't enough to determine conformance. I don't think this suggestion is accurate. It may also come across as contradicting the "they are not required for determining conformance" statement further down the page. Prefer to keep the text as is.

> - [medium-minor] Why the order: Test Methodologies, Semi-automated Test Tools, Automated Test Tools

People often think ACT is all about automated testing. I wanted to draw attention to manual and semi-automated testing by putting them higher on the page. They should be able to get higher numbers then automated tools too, since they're less constraint. This was discussed and agreed on by ACT TF.

> - [minor] copyedit ... 

I don't understand the request here.

> - [medium] email contact

I'll open a separate issue for that. This is not specific to this PR.

File: content/index.md [hunk 5]
```diff
@@ -11,37 +12,52 @@ github:
...
+{%- for rule in site.data.wcag-act-rules.rules["act-rules"] %}
+  {%- if rule.proposed == false %}
+    <li><a href="{{ rule.permalink | relative_url }}">{{ rule.title }}</a></li>
+  {%- endif %}
+{%- endfor %}
+</ul>
+
+## Proposed Rules for WCAG 2
+
+These ACT Rules are used to test conformance issues of WCAG 2.0 or 2.1. These rules will be considered for approval once they are fully implemented in at least one test tool or methodology.
```
[the remark this answers is not in the document]
Comment (daniel-montalvo) [reply content/index.md]: That  is clearer for me now. Thanks.

File: content/implementations.md [hunk 6]
```diff
@@ -20,36 +20,36 @@ github:
...
 Test methodologies provide step by step instructions on how to test accessibility standards.
-See [understanding ACT consistency](#understandingactconsistency) for details.
+See [understanding ACT consistency](#understanding-act-consistency) for details.
 
 {% assign methodologies = implementations | where: "type", "Test methodology" %}
 {% include_relative _implementations-table.html implementations=methodologies %}
 
 ## Semi-automated Test Tools
 
 Tools that combine user input and automated testing to test accessibility standards.
```
Comment (daniel-montalvo) [inline content/implementations.md:36]:
```suggestion
Semi-automated tools combine user input and automated testing to test accessibility standards.
```

This is to harmonize how we write these three similarparagraphs.

File: content/implementations.md [hunk 7]
```diff
@@ -20,36 +20,36 @@ github:
 </section>
 
 {::nomarkdown} {% include box.html type="start" title="Summary" %} {:/}
-The tables on these pages show how many ACT Rules different accessibility test tools and methodologies have consistently implemented. Each implementation links to a report with more details on how rules are implemented. See [understanding ACT consistency](#understandingactconsistency) for details.
+The tables on these pages show how many ACT Rules different accessibility test tools and methodologies have consistently implemented. Each implementation links to a report with more details on how rules are implemented. See [understanding ACT consistency](#understanding-act-consistency) for details.
```
Comment (daniel-montalvo) [inline content/implementations.md:23]:
```suggestion
The tables on these pages show how many ACT Rules different accessibility test tools and methodologies have "consistently implemented". Each implementation links to a report with more details on how rules are implemented. See [understanding ACT consistency](#understanding-act-consistency) for details.
```

This clarifies that it is a specific concept that we will be defining later.

**Review by daniel-montalvo — CHANGES_REQUESTED** [review]
Looking much better!

In addition to my suggestions, please consider keeping the "Draft" indicator until we have the data from the vendors that are currently listed as "coming soon".

File: _config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply _config.yml:71]: I think we can leave these as is now.
